apps/pie-storybook/stories/pie-cookie-banner.stories.ts

```diff
@@ -23,14 +24,17 @@ const keptReferences = [
 type CookieBannerStoryMeta = StoryMeta<CookieBannerProps>;
 
 const defaultArgs: CookieBannerProps = {
-
+    hasPrimaryActionsOnly: false,
 };
 
 const cookieBannerStoryMeta: CookieBannerStoryMeta = {
     title: 'Cookie Banner',
     component: 'pie-cookie-banner',
     argTypes: {
-
+        hasPrimaryActionsOnly: {
+            description: 'When true, sets the "Necessary only" button variant to "primary".',
```

Comment:
Same as below:

`When true, sets all action buttons in the cookie banner to display as variant "primary".`

Reply:
I actually think what I have is clearer, because "Manage preferences" is a button but that remains as an inverse button rather than changing to primary.

Reply:
Good point - I don't mind too much either way, just was wondering if it could be decoupled from the translation text